lisa/node.py

```diff
@@ -164,6 +164,9 @@ def create(
     def reboot(self) -> None:
         self.tools[Reboot].reboot()
 
+    def specific_timeout_reboot(self, time_out: int) -> None:
```

Comment:
It doesn't need to create another method, add an argument with default value like below.

```python
def reboot(self, timeout: int = 300) -> None:

```

Reply:
Changed